portfolio/src/main/java/com/google/sps/servlets/data/PersonalDetails.java

```diff
@@ -28,7 +27,9 @@ public int getCurrentAge() {
     return currentAge;
   }
 
-  public String getSchoolYear() {
-    return schoolYear;
+  public String getRandomFact() {
```

Comment:
Usually we want to add javadoc for every public function, but these function names are pretty self-explanatory